### ISSUE TITLE
OD position

### ISSUE THREAD

**Issue by hahabingo** [babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology#1]
hello, Is the location of optic disc in this article based on the selection of the highest point of pixel brightness? I find it didn't work to some images.

**Comment by babloogpb1** [comment]
Hii, The centre of od is not the brightest pixel location. If you refer to the paper, the authors actually mentioned a procedure to find the centroid of od using some distance function and a close hole operator. We didn't implement that part in the code, but we computed the centroid by considering the brightest region(od) and finding its centre.

**Comment by hahabingo** [comment]
Thank you very much for your reply. I got it. Have you implemented his method in your code now?&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"G Pavan Bhargav"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月3日(星期三) 下午2:34
收件人:&nbsp;"babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology"<Optic-disc-segmentation-using-pca-and-mathematical-morphology@noreply.github.com&gt;;
抄送:&nbsp;"1592404966"<1592404966@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology] OD position (#1)





 
Hii, The centre of od is not the brightest pixel location. If you refer to the paper, the authors actually mentioned a procedure to find the centroid of od using some distance function and a close hole operator. We didn't implement that part in the code, but we computed the centroid by considering the brightest region(od) and finding its centre.

**Comment by babloogpb1** [comment]
yes, we've used open-cv watershed function instead of the one given in the paper.

**Comment by hahabingo** [comment]
Thank you, I tested&nbsp; watershed.py and&nbsp;contour_only.py . I find that I get multiple centers and multiple coordinates, as shown in the pictures I sent. I am wondering if you've been doing any work in this field lately? Can you share the latest code with me? Thank you very much!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"G Pavan Bhargav"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月3日(星期三) 下午2:53
收件人:&nbsp;"babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology"<Optic-disc-segmentation-using-pca-and-mathematical-morphology@noreply.github.com&gt;;
抄送:&nbsp;"1592404966"<1592404966@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology] OD position (#1)





 
yes, we've used open-cv watershed function instead of the one given in the paper.

**Comment by babloogpb1** [comment]
oh, I'm afraid to say that I'm not working on it any more. The code in the repo is the latest one.

**Comment by hahabingo** [comment]
Glad&nbsp;to&nbsp;hear&nbsp;from&nbsp;you.&nbsp; Let&nbsp;me&nbsp;introduce&nbsp;myself.&nbsp;I'm&nbsp;from&nbsp;Suzhou,&nbsp;China. I&nbsp;am&nbsp;a&nbsp;graduate&nbsp;student&nbsp;in&nbsp;my&nbsp;first&nbsp;year. And welcome to visit SuZhou.
 

I&nbsp;will&nbsp;continue&nbsp;to&nbsp;do&nbsp;this&nbsp;research&nbsp;in&nbsp;the&nbsp;future,&nbsp;and&nbsp;I&nbsp;hope&nbsp;that&nbsp;we&nbsp;can&nbsp;have&nbsp;more&nbsp;opportunities&nbsp;to&nbsp;communicate.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;"G Pavan Bhargav"<notifications@github.com&gt;;
发送时间:&nbsp;2020年6月3日(星期三) 晚上10:37
收件人:&nbsp;"babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology"<Optic-disc-segmentation-using-pca-and-mathematical-morphology@noreply.github.com&gt;;
抄送:&nbsp;"1592404966"<1592404966@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [babloogpb1/Optic-disc-segmentation-using-pca-and-mathematical-morphology] OD position (#1)





 
oh, I'm afraid to say that I'm not working on it any more. The code in the repo is the latest one.

**Comment by babloogpb1** [comment]
Yeah. Glad to hear too. Thank you.
